Partition_2/doc/Partition_2/CGAL/Partition_traits_2.h

```diff
@@ -34,115 +36,79 @@ class Partition_traits_2 {
 /*!
 
 */ 
-typedef R::Line_2 Line_2; 
-
-/*!
-
-*/ 
-typedef R::Segment_2 Segment_2; 
-
-/*!
-
-*/ 
-typedef R::Ray_2 Ray_2; 
-
-/*!
-
-*/ 
-typedef R::Less_yx_2 Less_yx_2; 
-
-/*!
-
-*/ 
-typedef R::Less_xy_2 Less_xy_2; 
-
-/*!
+typedef R::FT FT; 
 
-*/ 
-typedef R::Left_turn_2 Left_turn_2; 
 
 /*!
 
 */ 
-typedef R::Orientation_2 Orientation_2; 
+typedef boost::property_traits<PointPropertyMap>::key_type Point_2; 
 
+    
 /*!
 
 */ 
-typedef R::Compare_y_2 Compare_y_2; 
+typedef std::list<Point_2> Container; 
```

Comment:
There is a rule stating that if you don't have a precise reason to use a container then use a vector. So except if you have insertion in the middle of a sequence you should use a vector.